scss/components/link.scss

```diff
@@ -10,3 +10,10 @@ $block: #{$fd-namespace}-link;
 .#{$block} {
   @include fd-link();
 }
+
+a.#{$block} {
+  font-size: $fd-font-size;
+  font-family: $fd-font-family;
+  line-height: $fd-line-height;
+  -webkit-font-smoothing: antialiased;
+}
```

Comment:
These styles are all in the `fd-reset` mixin.  You could remove this block and the change the block above to this:
```scss
.#{$block} {
    @include fd-reset();
    @include fd-link();
}
```